package.json

```diff
@@ -212,6 +214,7 @@
     "fs-extra": "^9.0.1",
     "handlebars": "^4.7.6",
     "http-proxy": "^1.18.1",
+    "jest-canvas-mock": "^2.3.0",
```

Comment:
What's it for?

> This should only be installed as a development dependency (devDependencies) as it is only designed for testing. (from npm)

Reply:
Some canvas errors appears in test log when opening a terminal tab inside test case (probably there is some xterm vs jest issues). This mock module cleans error output. Moving it to dev dependencies.